src/app-layout/visual-refresh-toolbar/navigation/styles.scss

```diff
@@ -28,6 +28,10 @@
     display: none;
   }
 
+  &.is-navigation-open > .content-container {
+    display: contents;
```

Comment:
When applying `display: contents` to an element, all other styles applied to this element are ignored

![Screenshot 2024-11-06 at 12 15 37](https://github.com/user-attachments/assets/9619bf85-bb53-446c-bc22-b1e0313ca2ea)

Can we clean up the currently unused styles (if they do nothing) or reassign them to different elements (if they still have some use)

Reply:
P.S. If `display: contents` does the job, it seems like we can simply remove that DOM node (because this is what `display: contents` effectively does)

Reply:
We need that node to prevent the text from scrolling in when the drawer is opened with motion. Updated to display flex instead of contents.